src/map/vana_time.cpp

```diff
@@ -196,8 +196,10 @@ TIMETYPE CVanaTime::SyncTime()
     m_vHour = (uint32)((m_vanaDate % VTIME_DAY)   / VTIME_HOUR);
     m_vMin  = (uint32)( m_vanaDate % VTIME_HOUR);
 
-    if (m_vMin == 0)
+    static int8 lastTickedHour = m_vHour;
+    if (m_vHour == (lastTickedHour+1)%24)
```

Comment:
Does this `if` still fire if we're assigning `lastTickedHour` to be `m_vHour` right before checking if the current hour is our (just-assigned) `lastTicked` hour plus one?

Reply:
Hey @ibm2431 ! It will, because I've declared it as static. Static variables are only initialized once the first time they're used, and will have indefinite lifespan like globals do, but they still maintain local scope. I selected static rather than a global variable since this variable never needs to be used outside of this local function. :)

Reply:
I actually had this same question lol. So the declaration happens once, then the reassignment happens every time?

Reply:
The declaration happens only once, and the assignment happens only once. Basically, after the first pass, line 199 is inert and does nothing. The only time it'll be assigned is after the following `if` statement is true, since I have an assignment statement in there.

Reply:
Yeah I meant the assignment on line 202. Makes sense, thanks for clarifying! Good work.